### PR TITLE
SendNotification Cleaned up

### PR DESCRIPTION
**Pull request by miralyousef**
- fixed sendNotification to take in List<Assignments> 
- added scheduled attribute to assignments

### CODE REVIEW

File: src/main/java/app/AppBuilder.java
Comment (Sahirara) [inline src/main/java/app/AppBuilder.java:1]: One of the code conflicts – should be easily fixed via Github

File: src/main/java/entity/Assignment.java
Comment (Sahirara) [inline src/main/java/entity/Assignment.java:1]: One of the code conflicts – should be easily fixed via Github

File: src/main/java/view/AssignmentAddView.java [hunk 1]
```diff
@@ -74,17 +74,13 @@ public void actionPerformed(ActionEvent evt) {
                         if (evt.getSource().equals(addAssignment)) {
                             final AddAssignmentState currentState = addAssignmentViewModel.getState();
 
-                            try {
```
Comment (Sahirara) [inline src/main/java/view/AssignmentAddView.java:77]: The removal of the try-catch here is causing code conflicts – see if you're able to fix this on your end or in Github

File: src/main/java/entity/CommonAssignment.java
Comment (AlishaZariff) [inline src/main/java/entity/CommonAssignment.java:1]: Missing documentation, overall clean coding style.

File: src/main/java/interface_adapter/send_notification/SendNotificationPresenter.java
Comment (AlishaZariff) [inline src/main/java/interface_adapter/send_notification/SendNotificationPresenter.java:1]: Proper interlinking of code components, documentation of expectations for prepare fail view. 
TODO remains to be completed.

File: src/main/java/interface_adapter/send_notification/SendNotificationState.java
Comment (AlishaZariff) [inline src/main/java/interface_adapter/send_notification/SendNotificationState.java:1]: State made according to clean architecture standards, amazing. TODO leaves some questions to be discussed for efficiency of the code.

File: src/main/java/interface_adapter/send_notification/SendNotificatonController.java
Comment (AlishaZariff) [inline src/main/java/interface_adapter/send_notification/SendNotificatonController.java:1]: Made according to clean architecture standards.

**Pull request merged**